views/components/song-controls.pug

```diff
@@ -0,0 +1,7 @@
+//- need to mess with this class name to display & no slider
+div(class="100-wid")
+    div(class='title-container active', id="current-song-display")
+        a(class='songDisplay') Current song: 
+        a(id='titleID', class='songDisplay')
+        a(class='songDisplay')  by 
+        a(id='artistID', class='songDisplay')
```

Comment:
Why are they `anchor` tags instead of `span`s?

Reply:
from last group, making ticket for now